claat/render/template.go

```diff
@@ -27,11 +27,12 @@ import (
 
 // Context is a template context during execution.
 type Context struct {
-	Env      string
-	Prefix   string
-	GlobalGA string
-	Meta     *types.Meta
-	Steps    []*types.Step
+	Env       string
+	Prefix    string
+	GlobalGA  string
+	Meta      *types.Meta
+	Steps     []*types.Step
+	ExtraVars map[string]string // Extra variables passed from the command line.
```

Comment:
Also, what about making it `map[string]interface{}`? That way you could pass strings, ints, floats, arrays and literally anything else that can be unmarshaled.